cms/core/utils.py

```diff
@@ -54,16 +58,53 @@ def get_formatted_pages_list(
             "description": getattr(page, "listing_summary", "") or getattr(page, "summary", ""),
         }
         if release_date := page.release_date:
-            datum["metadata"]["date"] = {
-                "prefix": _("Released"),
-                "showPrefix": True,
-                "iso": date_format(release_date, "c"),
-                "short": date_format(release_date, "DATE_FORMAT"),
-            }
+            datum["metadata"]["date"] = get_document_metadata_date(release_date, _("Released"))
         data.append(datum)
     return data
 
 
+def get_document_metadata_date(value: date | datetime | str, prefix: "StrOrPromise") -> dict[str, Any]:
+    """Returns a dictionary with formatted date information for the document metadata."""
+    return {
+        "prefix": prefix,
+        "showPrefix": True,
+        "iso": date_format(value, "c"),
+        "short": date_format(value, "DATE_FORMAT"),
+    }
+
+
+def get_document_metadata(
+    content_type: "StrOrPromise", date_value: date | datetime | str | None, prefix: "StrOrPromise"
+) -> dict[str, Any]:
+    """Returns a dictionary with formatted metadata information for the document."""
+    metadata = {
+        "object": {"text": content_type},
+    }
+
+    if date_value:
+        metadata["date"] = get_document_metadata_date(date_value, prefix)
+
+    return metadata
+
+
+def get_content_type_for_page(page: "Page") -> "StrOrPromise":
```

Comment:
This looks unused

Reply:
It's used here:

https://github.com/ONSdigital/dis-wagtail/blob/abcddf56dec9791b768daadec5afd5b31d93f821/cms/standard_pages/models.py#L132-L134

Reply:
huh, so it is! Somehow `ag` and my IDE search function failed miserably